erpnext/stock/doctype/shipment_parcel_template/shipment_parcel_template.json

```diff
@@ -0,0 +1,78 @@
+{
+ "actions": [],
+ "autoname": "field:preset_name",
+ "creation": "2020-07-09 11:43:43.470339",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "preset_name",
+  "length",
+  "width",
+  "height",
+  "weight"
+ ],
+ "fields": [
+  {
+   "fieldname": "preset_name",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Preset Name",
```

Comment:
Parcel Template Name would be better. Not everyone will understand the context of Preset